metadata/src/main/java/com/redhat/lightblue/metadata/FieldConstraint.java

```diff
@@ -31,4 +31,10 @@
      * Determines if the constraint is valid for the given field type
      */
     boolean isValidForFieldType(Type fieldType);
+
+
+    /**
+     * Returns the description of the FieldConstraint (for documentation propose)
+     */
+    String getDescription();
```

Comment:
It was used, by I just quickly  reviewed it before we talk about it, but I think it is just the regex case now, but it was used for other cases as well (when I had a different concept of the schema json that we would generate), so I can refactor it to something different just for regex case now if you think it would be better